### PR TITLE
Update lock styles

### LINKED ISSUE #1090: Purchase Button for Paywall Card
**Description**
When a user is served the paywall a card appears that indicates price of the key. The card is clickable but to make it more apparent we've added a Purchase button. The user can still click anywhere on the card to start the payment process

**Screenshot**
image

**Specs**
https://www.figma.com/proto/LkWSSnCGlKEITBB5T5NYXW/Unlock-Paywall?node-id=0%3A825&viewport=1269%2C738%2C0.355284&scaling=min-zoom

### LINKED ISSUE #1156: Missing duration on locks
When showing the lock on the paywall we currently do not show the duration of the purchased keys.
@smombartz Could you provide a quick desighn update?

image

### PR DESCRIPTION
**Pull request by cnasc**
# Description

This PR fixes #1156 and fixes #1090. It updates the various lock stages to use the new design proposed by @smombartz . Critique on this implementation, especially around the hover states for the ConfirmedKeyLock are most appreciated.

screen shot 2019-01-23 at 12 15 50 pm
screen shot 2019-01-23 at 12 16 08 pm
screen shot 2019-01-23 at 12 16 20 pm
screen shot 2019-01-23 at 12 16 29 pm
screen shot 2019-01-23 at 12 16 36 pm
screen shot 2019-01-23 at 12 16 44 pm
screen shot 2019-01-23 at 12 17 00 pm

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [x] My code follows the style guidelines of this project, including naming conventions
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

File: unlock-app/src/components/interface/buttons/overlay/ConfirmedKey.js [hunk 1]
```diff
@@ -15,20 +16,27 @@ ConfirmedKey.propTypes = {
   hideModal: PropTypes.func.isRequired,
 }
 
-const NotHover = styled(Svg.Checkmark)``
+export const NotHover = styled(Svg.Checkmark)``
```
Comment (julien51) [inline unlock-app/src/components/interface/buttons/overlay/ConfirmedKey.js]: I think you are not actually using these? In that case, maybe the export statement is superfluous?

File: unlock-app/src/components/lock/ConfirmedKeyLock.js [hunk 2]
```diff
@@ -28,7 +42,7 @@ ConfirmedKeyLock.propTypes = {
 
 export default ConfirmedKeyLock
 
-const Header = styled(LockHeader)`
-  background-color: var(--green);
-  color: var(--offwhite);
+const Body = styled(LockBody)`
+  border: 1px solid var(--green);
+  padding-top: 0;
```
Comment (julien51) [inline unlock-app/src/components/lock/ConfirmedKeyLock.js]: I think this `padding-top: 0;` is used on all the `Body` below? Maybe worth moving to `LockBody`?

File: unlock-app/src/components/lock/ConfirmingKeyLock.js [hunk 3]
```diff
@@ -44,7 +46,13 @@ ConfirmingKeyLock.propTypes = {
 
 export default withConfig(ConfirmingKeyLock)
 
-const Header = styled(LockHeader)`
-  background-color: var(--green);
-  color: var(--offwhite);
+const Body = styled(LockBody)`
+  border: 1px solid var(--yellow);
+  padding-top: 0;
```
Comment (julien51) [inline unlock-app/src/components/lock/ConfirmingKeyLock.js]: ditto

File: unlock-app/src/components/interface/buttons/overlay/ConfirmedKey.js (hunk 1 above)
Comment (cnasc) [reply unlock-app/src/components/interface/buttons/overlay/ConfirmedKey.js]: Oops, accidentally left that in from when I tried a few different ways to get the hover. Good catch.

File: unlock-app/src/components/lock/ConfirmedKeyLock.js (hunk 2 above)
Comment (cnasc) [reply unlock-app/src/components/lock/ConfirmedKeyLock.js]: Yes, the only one that has a different value is `NoKeyLock`, so I will move the `padding-top: 0;` to `LockBody` and then override it for `NoKeyLock`.

**Pull request merged**